### PR TITLE
#1070 replacing Reshape and Transpose operators to the kernel that are  invalid in tensorrt with just a reshaped tensor

### PR DESCRIPTION
**Pull request by NikolasMarkou**
I replaced the reshape operator for the kernel with a numpy reshape of the kernel.

### CODE REVIEW

File: tf2onnx/onnx_opset/nn.py [hunk 1]
```diff
@@ -124,17 +124,12 @@ def conv_convert_inputs(ctx, node, with_kernel=False, new_kernel_shape=None,
...
-                shape_name = utils.make_name(node.name)
-                ctx.make_const(shape_name, np.array(new_kernel_shape, dtype=np.int64))
-
-                reshape = ctx.make_node("Reshape", [kernel_name, shape_name])
-                ctx.replace_input(node, kernel_name, reshape.output[0], 1)
-
-                reshape.skip_conversion = True
-
-            ctx.set_shape(reshape.output[0], new_kernel_shape)
+                input_node = node.inputs[1]
```
[the remark this answers is not in the document]
Comment (TomWildenhain-Microsoft) [reply tf2onnx/onnx_opset/nn.py:121]: Nice.  Why is the `len(node.inputs) == 2` check needed?

[the remark this answers is not in the document]
Comment (NikolasMarkou) [reply tf2onnx/onnx_opset/nn.py:121]: @TomWildenhain-Microsoft would it be more general to set it as 
```len(node.inputes) >= 2```

[the remark this answers is not in the document]
Comment (TomWildenhain-Microsoft) [reply tf2onnx/onnx_opset/nn.py:121]: I think it is always true so a check isn't needed.  The old code at line 120 accesses node.input[1] so it must be safe to assume node.inputs has length at least 2.  Or is there some other reason you added that check?

File: tests/test_backend.py [hunk 2]
```diff
@@ -3565,6 +3566,27 @@ def func(y, x):
...
+                                dilations=dilations)
+            return tf.identity(conv, name=_TFOUTPUT)
+
+        self._run_test_case(func, [_OUTPUT], {_INPUT: x_val, _INPUT2: w_val}, rtol=rtol)
+
+    def test_conv2d_1_kernel_as_input(self):
+        x_val = make_xval((1, 1, 5, 5)).transpose(NCHW_TO_NHWC)
+        w_val = np.array([[2., 1., 1.],
+                      [1., 3., 1.],
+                      [1., 1., 4.]], dtype=np.float32).reshape(_KERNEL3x3)
```
Comment (TomWildenhain-Microsoft) [inline tests/test_backend.py:3587]:
```suggestion
        w_val = np.array([[2., 1., 1.],
                          [1., 3., 1.],
                          [1., 1., 4.]], dtype=np.float32).reshape(_KERNEL3x3)
```
pylint

File: tf2onnx/onnx_opset/nn.py (hunk 1 above)
[the remark this answers is not in the document]
Comment (NikolasMarkou) [reply tf2onnx/onnx_opset/nn.py:121]: No other reason really

**Pull request merged**